### PR TITLE
Allow handler to use commands on application returned to applicant

### LINKED ISSUE #1358: Do not disable handler actions when application is returned to applicant
Handler often needs to close stale application, and spirit of REMS is that handler can do anything to applications anyway. Only drafts should be hidden (as those are just noise for handlers).

### PR DESCRIPTION
**Pull request by okahilak**
Closes #1358

### CODE REVIEW

File: src/clj/rems/application/model.clj [hunk 1]
```diff
@@ -18,82 +18,76 @@
...
+   :application.command/invite-member
+   :application.command/uninvite-member
+   :application.command/request-comment
+   :application.command/request-decision
+   :application.command/return
+   :application.command/approve
+   :application.command/reject])
+
+(def ^:private handler-returned-commands
+  (vec (remove #(= % :application.command/return) handler-all-commands)))
```
Comment (luontola) [inline src/clj/rems/application/model.clj]: handler-all-commands could be a set, which would simplify this code to just a disj

Comment (okahilak) [reply src/clj/rems/application/model.clj]: A nice point, done. Although it adds some inconsistency between handler-all-commands and, e.g., submittable-application-commands.

File: src/clj/rems/application/model.clj [hunk 2]
```diff
@@ -18,82 +18,74 @@
...
-             :application.command/remove-member
-             :application.command/invite-member
-             :application.command/uninvite-member
-             :application.command/request-comment
-             :application.command/request-decision
-             :application.command/return
-             :application.command/approve
-             :application.command/reject]
+  {:applicant non-submittable-application-commands
+   :handler (vec (conj handler-all-commands :see-everything))
```
Comment (luontola) [inline src/clj/rems/application/model.clj]: These vec calls are not needed. update-role-permissions doesn't care what kind of a collection the parameter is.

Comment (okahilak) [reply src/clj/rems/application/model.clj]: Ok, should I then change all these vectors to sets for consistency?

File: src/clj/rems/application/model.clj (hunk 1 above)
Comment (luontola) [reply src/clj/rems/application/model.clj]: For consistency you could convert them all to sets.

File: src/clj/rems/application/model.clj (hunk 2 above)
Comment (luontola) [reply src/clj/rems/application/model.clj]: Yes, sounds good.

Comment (okahilak) [reply src/clj/rems/application/model.clj]: Done.

**Pull request merged**